tests/Sinch.Tests/e2e/Verification/VerificationStatusTests.cs

```diff
@@ -88,8 +88,8 @@ public async Task ByIdentityPhoneCall()
                 await VerificationClient.VerificationStatus.GetByIdentity("+49342432",
                     VerificationMethod.Callout);
```

Comment:
The test name needs to be updated https://github.com/sinch/sinch-sdk-dotnet/pull/40/files#diff-8e74407e5b46b5488b4e3b52658cc55f36143ac5c8a7d1d5b6b40ccbeef6bbb2R85

Reply:
That's for verification status, which will be a separate PR